launch/dynamo-run/src/lib.rs

```diff
@@ -344,3 +364,39 @@ async fn stopper(
     // Keep it alive until the engine has stopped.
     drop(py_script);
 }
+
+/// If the user will benefit from CUDA/Metal/Vulkan, remind them to build with it.
+/// If they have it, celebrate!
+// Only mistralrs and llamacpp need to be built with CUDA.
+// The Python engines only need it at runtime.
+#[cfg(any(feature = "mistralrs", feature = "llamacpp"))]
```

Comment:
Question: what if we made cuda , metal, vulkan on by default (depending on the environment)?

So opt out instead of opt in if the dependencies are available ... may not be possible to determine environment.

Reply:
`cuda`, `metal` and `vulkan` have to be set at build time. They are also exclusive, I don't think we can enable more than one.

Currently the build we release is CPU-only, and I would really like us to get CUDA as the default by 0.3.1: https://github.com/ai-dynamo/dynamo/issues/1217